src/settings_gui.cpp

```diff
@@ -763,7 +779,7 @@ static const NWidgetPart _nested_game_options_widgets[] = {
 				EndContainer(),
 
 				NWidget(WWT_FRAME, COLOUR_GREY), SetDataTip(STR_GAME_OPTIONS_CURRENCY_UNITS_FRAME, STR_NULL),
-					NWidget(WWT_DROPDOWN, COLOUR_GREY, WID_GO_CURRENCY_DROPDOWN), SetMinimalSize(100, 12), SetDataTip(STR_JUST_STRING, STR_GAME_OPTIONS_CURRENCY_UNITS_DROPDOWN_TOOLTIP), SetFill(1, 0),
+					NWidget(WWT_DROPDOWN, COLOUR_GREY, WID_GO_CURRENCY_DROPDOWN), SetMinimalSize(100, 12), SetDataTip(STR_JUST_STRING3, STR_GAME_OPTIONS_CURRENCY_UNITS_DROPDOWN_TOOLTIP), SetFill(1, 0),
```

Comment:
Isn't `STR_JUST_STRING2` enough? You have a string with 2 (additional) parameters.
`STR_JUST_STRING(0)` needs `SetDParam(0, STR_something)`. `STR_JUST_STRING1` then has 1 extra parameter, so sets parameter 0 to the string ID and parameter 1 is the first parameter to that string.

Reply:
Hmm, not sure, parameters 0, 1, and 2 are used for WID_GO_CURRENCY_DROPDOWN, which can be either just the currency name (for custom or NewGRF-overridden currencies), or for the STR_GAME_OPTIONS_CURRENCY_CODE string code which takes additional currency name and currency code as parameters.